spatialpandas/geometry/base.py

```diff
@@ -536,9 +536,15 @@ def fillna(self, value=None, method=None, limit=None):
     @property
     def sindex(self):
         if self._sindex is None:
-            self._sindex = HilbertRtree(self.bounds)
+            self.build_sindex()
         return self._sindex
 
+    def build_sindex(self):
+        if self._sindex is None:
+            self._sindex = HilbertRtree(self.bounds)
+        else:
+            raise ValueError("Spatial index has already been built.")
+
```

Comment:
My intuition is that building the spatial index multiple times should be a no-op. Are there any problems you see with doing nothing in this case without raising an error?

Reply:
True, happy to make it a no-op.